bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -776,18 +769,18 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!--
      clm6_0 physics with CAM7.0 grids...
 -->
-<!-- These first five are for SP mode and CLM6.0 only -->
-<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="1979" lnd_tuning_mode="clm6_0_cam7.0"
-                  maxpft="17"        use_cn=".false."    use_crop=".false." hgrid="0.9x1.25"            >.true.</use_init_interp>
-<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="1979" lnd_tuning_mode="clm6_0_cam7.0"
-                  maxpft="17"        use_cn=".false."    use_crop=".false." hgrid="1.9x2.5"             >.true.</use_init_interp>
+<!-- These are for 1979 for f09 and f19 for any atmospheric forcing -->
+<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="1979" phys="clm6_0"
+                  hgrid="0.9x1.25"            >.true.</use_init_interp>
+<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="1979" phys="clm6_0"
+                  hgrid="1.9x2.5"            >.true.</use_init_interp>
+<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="1979" phys="clm6_0"
+                  hgrid="1.9x2.5"            >.true.</use_init_interp>
```

Comment:
These two are identical. Was one intended to be different?

Reply:
I'm removing the redundant lines. We can address it later if the intention was different.

Reply:
Yeah it's a good question. That I'll evaluate and maybe we'll do something later.